types/forge-viewer/index.d.ts

```diff
@@ -1115,6 +1116,50 @@ declare namespace Autodesk {
             function getHtmlTemplate(url: string, callback: (error: string, content: string) => void): void;
             function lerp(x: number, y: number, t: number): number;
 
+            class FragmentList {
+              allVisible: boolean;
+              allVisibleDirty: boolean;
+              animxforms: any;
+              boxes: any;
+              db2ThemingColor: any[];
+              dbIdIsGhosted: any[];
+              dbIdOpacity: any[];
+              fragments: any;
+              geomids: Int32Array;
+              geoms: GeometryList;
+              is2d: boolean;
+              isFixedSize: boolean;
+              linesHidden: boolean;
+              materialIdMap: { [id: number]: any };
+              materialids: Int32Array;
+              materialmap: any;
+              matrix: any;
+              modelId: number;
+              nextAvailableFragID: number;
+              nextMaterialId: number;
+              originalColors: any[];
+              pointsHidden: boolean;
+              themingOrGhostingNeedsUpdate: any[];
+              transforms: any;
+              useThreeMesh: boolean;
+              vizflags: Uint32Array;
+              vizmeshes: THREE.Mesh[];
+            }
+
+            class GeometryList {
```

Comment:
Similar question as with `FragmentList`, above. Is `GeometryList` a value exported by the namespace `Private`, or should this be declared as an `interface`?

Reply:
Same as above.